<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="B-Windows10-Sel" name="Test" tests="1" failures="1" timestamp="11 Jan 2017 14:39:57 GMT" time="11.057" errors="0">
  <testcase name="AddNoDiscount" time="3.482" classname="testcases.AddEmployee$AddEmployeeTest">
    <failure type="java.lang.AssertionError" message="Error found in dashboard data after adding employee without discount expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Error found in dashboard data after adding employee without discount expected [true] but found [false]
	at testcases.AddEmployee$AddEmployeeTest.AddNoDiscount(AddEmployee.java:17)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- AddNoDiscount -->
</testsuite> <!-- Test -->
